frameworks/projects/Jewel/src/main/royale/org/apache/royale/jewel/beads/controls/combobox/ComboBoxItemByField.as

```diff
@@ -1,21 +1,65 @@
-package com.iest.winplusweb.beads
+////////////////////////////////////////////////////////////////////////////////
+//
+//  Licensed to the Apache Software Foundation (ASF) under one or more
+//  contributor license agreements.  See the NOTICE file distributed with
+//  this work for additional information regarding copyright ownership.
+//  The ASF licenses this file to You under the Apache License, Version 2.0
+//  (the "License"); you may not use this file except in compliance with
+//  the License.  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+package org.apache.royale.jewel.beads.controls.combobox
 {
   import org.apache.royale.core.IBead;
   import org.apache.royale.jewel.ComboBox;
   import org.apache.royale.core.IStrand;
   import org.apache.royale.collections.CollectionUtils;
   import org.apache.royale.collections.ArrayList;
   import org.apache.royale.events.Event;
+  
+  /**
+	 *  The ComboBoxItemByField class is a specialty bead that can be used with
+	 *  any ComboBox control. This bead allows to select an item by field
+	 *  
+	 *  @langversion 3.0
+	 *  @playerversion Flash 10.2
+	 *  @playerversion AIR 2.6
+	 *  @productversion Royale 0.9.4
```

Comment:
Royale version should be the next released one. Unfortunately, we still don't know what the next version number will be... I suggest using 0.9.8 because that has been the convention so far.

Reply:
Ok, perfect!!.